config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.yml

```diff
@@ -19,6 +19,7 @@ Limber-Htp - scRNA Core cDNA Prep GEM-X 5p:
 Limber-Htp - scRNA Core cDNA Prep GEM-X 5p GEMs Input:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_cdna_prep_gem_x_5p_gems_input"]
+    request_type_keys:
+      ["limber_scrna_core_cdna_prep_gem_x_5p_gems_input", "limber_scrna_core_cdna_prep_gem_x_5p_gems_input_cite"]
```

Comment:
Think this is wrong - you've removed the new request type, but it's still in the list here